tests/Microsoft.Data.DataFrame.Tests/DataFrameTests.cs

```diff
@@ -23,40 +23,80 @@ public static DataFrame MakeDataFrameWithTwoColumns(int length)
             return dataFrame;
         }
 
-        public static DataFrame MakeDataFrameWithAllColumnTypes(int length)
+        public static DataFrame MakeDataFrameForGroupBy(int length) => MakeDataFrameWithAllColumnTypes(length, true);
```

Comment:
Just adding this for now. As I add more functions to the GroupBy class, this may set up a DataFrame with edge case-y data

Reply:
Let's not introduce dead code into our tests. I don't see this method called anywhere.

Also having a `bool forGroupByTests` doesn't feel very sustainable. When the next test that comes along and needs some special test data, adding a new bool to these methods isn't going to work well.
Instead, it may be better to have common code in small reusable methods that are called from higher-level methods.

---
In reply to: [301800494](https://github.com/dotnet/corefxlab/pull/2696#discussion_r301800494) [](ancestors = 301800494)